unit-files/check_health.timer

```diff
@@ -0,0 +1,10 @@
+# check_health.timer
+[Unit]
+Description=Checks Bitcore performance health every minute
+
+[Timer]
+OnCalendar=minutely
```

Comment:
I would increase this a bit, since it can take bitcore over a minute to fully start. i would say 5-10 minutes would work better here